src/sentry/api/endpoints/group_tombstone.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+
+from sentry.models import GroupTombstone
+
+
+class GroupTombstoneEndpoint(ProjectEndpoint):
+    # TODO (katie): figure out correct permissions
```

Comment:
probably not, at least for now. i guess the only question is would `ProjectEventPermission` be better than `ProjectPermission`?